cypress.config.ts

```diff
@@ -1,6 +1,20 @@
 import { defineConfig } from 'cypress';
+import dotenv from 'dotenv';
+
+dotenv.config({
+  path: `.env`,
+});
+dotenv.config({
+  path: `.env.local`,
+  override: true,
+});
```

Comment:
I would cherry pick that dotenv calls to own file, because in future we might need to re-use that dotenv config.

Reply:
NextJS manages .env.* files on its own. Only cypress requires dotenv in this setup file. If it happens that we would need this somewhere else, we may create a separate module in the future.